ynr/apps/parties/tests/test_models.py

```diff
@@ -24,17 +24,16 @@ def test_party_emblem(self):
         party = PartyFactory()
         PartyEmblemFactory.create_batch(3, party=party)
         self.assertEqual(party.emblems.count(), 3)
-        print(party.emblems.first().image.url)
-        self.assertTrue(
-            party.emblems.first().image.url.startswith(
-                "{}/emblems/PP0/0_example".format(settings.MEDIA_ROOT)
-            )
+        first_emblem = party.emblems.first()
+        print(first_emblem.image.url)
```

Comment:
Can you remove this print?

Reply:
Great spot, removed in my most recent push :)